certification/results.go

```diff
@@ -1,20 +1,20 @@
-package runtime
+package certification
 
 import (
 	"time"
 
-	"github.com/redhat-openshift-ecosystem/openshift-preflight/certification"
+	"github.com/redhat-openshift-ecosystem/openshift-preflight/internal"
 )
 
 type Result struct {
-	certification.Check
+	internal.Check
 	ElapsedTime time.Duration
 }
 
 type Results struct {
 	TestedImage       string
 	PassedOverall     bool
-	TestedOn          OpenshiftClusterVersion
+	TestedOn          string
```

Comment:
I'm not sure why we changed this implementation detail, and added a `String()` method in runtime.go. Does this buy us something I'm not seeing?

Reply:
Also, this doesn't seem to work. This is always `unknown/unknown` in the json that gets written to disk, even though internal results struct is correct. I assume it's the genericFormatter that's causing the issue, but didn't look further.

Reply:
> I'm not sure why we changed this implementation detail, and added a `String()` method in runtime.go. Does this buy us something I'm not seeing?

There was no reason to expose the OpenshiftClusterVersion struct. I'll admit that where it stands right now is completely naive and may (probably?) doesn't work as it stands. I haven't looked at it further either.

Reply:
We do 'expose' OpenshiftClusterVersion struct though, in the sense that it is written to disk, and written to the console. And as it stands that does not work for the reasons I stated above.